src/coprocessor/dag/batch/executors/util/mock_executor.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use tipb::expression::FieldType;
+
+use crate::coprocessor::dag::batch::interface::*;
+
+/// A simple mock executor that will return batch data according to a fixture without any
+/// modification.
+///
+/// Normally this should be only used in tests.
+pub struct MockExecutor {
```

Comment:
Could we move it to the mod `test`?

Reply:
I think the name "MockExecutor" already clearly indicates that it is a test facility so that it doesn't need to be "test::MockExecutor".